experiment/find_developers.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Need to figure out why this only fails on travis
+# pylint: disable=bad-continuation
+
+
+"""Selects a random sample of kubernetes developers."""
+
+import json
+import os
+import random
+import sys
+
+import requests
+
+def download_content():
+    """Downloads contributor data from github."""
+    resp = requests.get('https://api.github.com/repos/kubernetes/kubernetes/stats/contributors')
+    resp.raise_for_status()
+    data = resp.content
+    return data
+
+
+def load_content(data):
+    """Parse the json response."""
+    users = [User(b) for b in json.loads(data)]
+    return users
+
+
+class User(object):  # pylint: disable=too-few-public-methods
+    """Store .user and number of .total and .recent commits."""
+    def __init__(self, blob):
+        self.user = blob['author']['login']
+        weeks = blob['weeks']
+        self.recent = sum(k['c'] for k in weeks[-12:])
+        self.total = sum(k['c'] for k in weeks)
+
+    def __cmp__(self, other):
```

Comment:
This could also be `def key(self): return self.recent, self.total, self.user` and then use sorted(load_content, key=User.key)`